### DISCUSSION TITLE
Mod selected in the beatmap selection screen affects the data in the beatmap detail interface

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#18840] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34775378/175763558-0468d856-9f50-4230-a7b0-43057ba8b35c.png)
![image](https://user-images.githubusercontent.com/34775378/175763449-2eb2e054-293c-4f5f-90e0-0cdb1313eee7.png)

I guess this shouldn't happen

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/18842

**Comment by respektive** [comment]
I don't know, I use this quite a lot when playing playlists, cause afaik there is no other way to see how mods affect the stats?

**Reply by frenzibyte** [reply]
Open a feature request for that. The beatmap info overlay is supposed to be completely decoupled from the game's current state, mainly including the currently selected mods.
